frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -178,7 +177,7 @@ class BitBox02 extends Component<Props, State> {
             const restoreSidebar = status === 'initialized' && !['createWallet', 'restoreBackup'].includes(appStatus) && sidebarStatus !== '';
             if (restoreSidebar) {
                 setSidebarStatus('');
-            } else if (status === 'connected' || (status !== 'initialized' && ['', 'forceCollapsed'].includes(sidebarStatus))) {
+            } else if (status === 'connected' || (status !== 'initialized' && ['', 'forceCollapsed'].includes(sidebarStatus))) {
```

Comment:
this looks like the same line changed in the previous commit.

Reply:
I accidentally added a werid white space again

Reply:
Should be gone with https://github.com/digitalbitbox/bitbox-wallet-app/compare/32de3278444ebb540c3fc0ec003ba834f29f82e6..49856c1a2b43b623e72781cada59848194f1941d